docs/technical_reference/flowsheets/ion_exchange.rst

```diff
@@ -0,0 +1,124 @@
+    continues with the optimization demonstration.
+    
+5. Optimally optimizing IX design with ``optimize_system``: 
+    Though not required, this function provides an example of optimizing the design of the IX system for a targeted effluent conenctration
+    of 25 mg/L. The initial model demonstration resulted in an effluent concentration of 0.21 mg/L, so this optimization results in longer breakthrough time
+    and therefore, a less frequent regeneration schedule and lower operating costs. The optimization proceeds in four steps:
+        
+    * First, the function will set an ``Objective`` on the flowsheet to minimize the levelized cost of water (LCOW).
+    * Then, the model fixes the effluent concentration of the IX model to 25 mg/L, propagates that concentration to the product block, and re-initializes the product block with the new targeted concentration.
+    * Next, three variables are unfixed on the ion exchange model to allow for the model to solve for the new conditions (``dimensionless_time``, ``number_columns``, and the ``bed_depth``).
+    * Finally, the model is solved for these new conditions.
+    
+    Note that stopping at this point will likely result in a non-integer value for ``number_columns``. This value also corresponds to an optimal value for both ``bed_depth`` and 
+    ``dimensionless_time``. The user can optionally ensure the ``number_columns`` is an integer by rounding the ``number_columns`` to an integer value after 
+    the initial optimization solve, fixing ``number_columns`` to that value, and then re-solving the model to obtain the final values for ``bed_depth`` and ``dimensionless_time``.
+    This is the approach taken in the demonstration file.
+        
+
+The following modeling components are used within the flowsheet:
+
+Documentation for property models:
+    * :doc:`/technical_reference/property_models/mc_aq_sol`
+Documentation for unit models:
+    * :doc:`/technical_reference/unit_models/ion_exchange_0D`
+Documentation for unit models from IDAES:
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/feed`
+    * :doc:`idaes:reference_guides/model_libraries/generic/unit_models/product`
+Documentation for costing models:
+    * :doc:`/technical_reference/costing/watertap_costing`
+    * :doc:`/technical_reference/costing/ion_exchange`
+
+Degrees of Freedom
+------------------
+
+The degrees of freedom for the flowsheet can change depending on the configuration options specified during the build. 
+Specifics for other configurations options are available in the IX model documentation.
+In this demonstration, the following variables are initially fixed for simulating the IX flowsheet (i.e., degrees of freedom = 0):
+
+    * feed conditions (component flows, temperature, pressure)
+    * Langmuir equilibrium coefficient for target ion
+    * Resin capacity, diameter, porosity, and density
+    * Number of columns for the system
+    * Service flow rate
```

Comment:
Perhaps this is a common term when discussing ion exchange, but I'm not exactly sure what it means. So I think it may be worth putting a short note in parentheses or rewording this to be a bit more clear what this flow rate is and how it differs from the previous feed conditions